server/views/proposedAddress/rejected.pug

```diff
@@ -33,8 +33,9 @@ block content
                 p.smallMarginBottom Reason for rejection:
                 if address.consent === 'No'
                     p.bold The homeowner/landlord does not give informed consent
+                // TODO check this
```

Comment:
have you checked this :point_up: .  If it should still be there then put your name against it in the TODO